docusaurus.config.js

```diff
@@ -7,7 +8,7 @@ const config = {
   url: 'https://surrealdb.com',
   // Set the /<baseUrl>/ pathname under which your site is served
   // For GitHub pages deployment, it is often '/<projectName>/'
-  baseUrl: '/',
+  baseUrl: '/docs/',
```

Comment:
This is now causing the Netlify preview to fail. We can check if `isProd` is true, but would it be better to configure Netlify to act more like the production environment, rather than change the setup for each environment?

Reply:
The URLs on prod & dev need to match because all the links in all the documents are relative to the browsers url. THat would mean adding a isprod check in thousands of places. I'm not sure why the preview is not working but I'll check it out. The file in the root is probably a placeholder file by docusaurus